src/shared/addons/addonapi.cpp

```diff
@@ -0,0 +1,149 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This is marked as a new file, but it's not really new, it's just moved. Can you fix this? Otherwise we lose the git history for it.